app/jobs/commit_patient_changesets_job.rb

```diff
@@ -33,7 +33,7 @@ def perform(import)
   end
 
   def import_patients_and_parents(changesets, import)
-    patients = changesets.map(&:patient)
+    patients = changesets.map(&:patient).uniq
```

Comment:
I think `deduplicate_patients!` is designed to do the same thing, although I'm not sure what that does which is different to `uniq`.